binaries/scootcl/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/scootdev/scoot/local/client/cli"
+	"github.com/scootdev/scoot/local/client/conn"
+	"log"
+	"os"
+)
+
+// A Scoot command-line client
+func main() {
+	dialer, err := conn.UnixDialer()
```

Comment:
Same here, if != nil err needs to return an exit code otherwise nil pointer exceptions or it'll use garbage.